xml/obs_ag_spider_identification.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<section id="_obs_spider_identification">
+ <title>Spider Identification</title>
+ <para>
+   OBS identifys spiders by using the gem <command>voight_kampff</command>. Spiders will be identified to optimize the
+   page content for them.
+ </para>
+ <para>
+   To update the spider list, you must run the command <command>rake voight_kampf:import_user_agents</command>.
```

Comment:
I cannot comment the first comment of you - for some reason. Anyways, I agree to your first comment.

To the second comment: There is no real documentation for this gem. The documentation links either pointing to the GitHub repository or to rubydoc.org what's just a collection of methods and classes in that gem. I would not like to point the admin to the README.md of this Git repository, for several reasons:
* The README.md describes it without any examples and in 3-4 sentences like "run xyz", "there is your config", "edit it", "regex is possible".
* The README.md can always change
* This guide describes it more concrete
* We don't know if the GitHub repository will get moved to some other platforms like GitBucket.

I asked one of the documentation team and she says that it's maybe better to just keep the documentation as it is for the reasons I posted above.